WebApp/autoreduce_webapp/templates/run_summary.html

```diff
@@ -13,29 +13,40 @@
             <div class="col-md-12 text-center">
                 <h2>Reduction Job #{{ run.title }}</h2>
                 {% if history.count > 1 %}
-                    (This run has been re-reduced, <a href="#" class="js-reduction-run-history" id="run_history">see history</a>)
+                    <p>
+                        (This run has been re-reduced,
+                        <a href="#" class="js-reduction-run-history" id="run_history">see history</a>)
```

Comment:
Does this still produce a space between `re-reduced,` and `see history`? Or will it appear as:
_(This run has been re-reduced,see history)_

Reply:
Yup, still there 👍